idlharness.js

```diff
@@ -444,156 +444,320 @@ IdlArray.prototype.assert_type_is = function(value, type)
      * is fairly elaborate due to the complexity of WebIDL's types, so it's
      * best to look at the grammar to figure out what properties it might have.
      */
+    var ret = this.check_type_is(value, type);
+    if (ret === true)
+    {
+        assert_true(true, "Passed");
```

Comment:
I thought that tests that ran no assertions were a problem of some kind.  I guess in wpt they're not right now, so you're right, this is pointless.  (In Gecko we eventually made mochitests that run no assertions fail, because it usually means a test bug.  It caught a number of tests that had syntax errors etc.)